spec/factories/interests.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :interest do
+    name 'Ruby'
+    type 'Work'
```

Comment:
It's a good idea to have all factories working without specifying any parameters - in that way, you can always use them by overriding only those parameters, that are important to the specific test.
But here it will fail because of the missing `user_id`.